go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/0chain/zwalletcli
 go 1.20
 
 require (
-	github.com/0chain/gosdk v1.11.7
+	github.com/0chain/gosdk v1.11.4-0.20240118170536-41dc66bf45b7
```

Comment:
Added older version just to test!

I'll update the tag once the go-sdk code is merged!

Reply:
Updated back to latest version